core/src/validator.rs

```diff
@@ -1507,7 +1594,12 @@ impl Validator {
         );
 
         *start_progress.write().unwrap() = ValidatorStartProgress::Running;
-        key_notifies.push(connection_cache);
+        if let Some(client_updater) = client_updater {
+            key_notifies.push(client_updater);
+        } else {
+            // add connection_cache because it is still used in Forwarder.
+            key_notifies.push(connection_cache);
```

Comment:
doesn't this need to be added even if tpu_client_next is used for STS, or the
forwarder identity won't change?